test/proper_tests.erl

```diff
@@ -1398,6 +1398,20 @@ sampleshrink_test_() ->
         ?_shrinksTo([a], Gen)},
        ?_test(proper_gen:sampleshrink(Gen))]}].
 
+existing_atom_test() ->
```

Comment:
It's very unclear what these tests _really_ test and what their underlying assumptions are.
For example, their correctness _crucially_ depends on **not**  loading any other Erlang modes between the two calls to `erlang:system_info(atom_count)`.  Is this robust to e.g. executing all the PropEr tests in parallel?

Also, it's unclear why the `default_atom_test` is using the `any()` instead of the `atom()` generator.  For example, the test trivially succeeds if `any()` happens to return terms other than atoms.